src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -312,8 +312,7 @@ def forward(
             present = None
 
         # [batch_size * num_heads, q_length, kv_length]
-        matmul_result = torch.baddbmm(
-            input=alibi,
+        matmul_result = alibi.baddbmm(
```

Comment:
This is stupid, but torch script works for `torch.Tensor.baddbmm` but not `torch.baddbmm` in torch v1.11.

Reply:
Make sure to put a comment stating that so that no one tries to change it the other way round.